### ISSUE TITLE
Do not show received payload in Jenkins job

### ISSUE THREAD

**Issue by ddimitrioglo** [jenkinsci/generic-webhook-trigger-plugin#32]
How should I configure `generic-webhook-trigger-plugin` to avoid showing 

```
Generic Cause
GenericWebhookEnvironmentContributor Received:
{
  // my request data here, I would like to hide it
}
```

**Comment by tomasbjerre** [comment]
Adding checkboxes for this now.

**Comment by ddimitrioglo** [comment]
@tomasbjerre You are cool, looking forward!

**Comment by tomasbjerre** [comment]
Released in 1.24 now.

**Comment by ddimitrioglo** [comment]
Confirmed! 
![image](https://user-images.githubusercontent.com/9005667/35858260-8094ac0a-0b44-11e8-9811-57c342932bfd.png)
